demo/osort-mem-trace/sort_A.cc

```diff
@@ -1,4 +1,4 @@
-#include "../../enclave/src/common/obl_primitives.h"
+#include "../../include/xgboost/common/obl_primitives.h"
```

Comment:
Can we avoid using relative paths here? i.e. changing this to `#include <xgboost/common/obl_primitives.h>`?

Reply:
Done.